imagetest/test_suites/packagevalidation/windows_version_test.go

```diff
@@ -235,23 +236,37 @@ func TestServicesState(t *testing.T) {
 		services = append(services, "google_osconfig_agent")
 
 	}
-
-	for _, service := range services {
-		out, err := utils.RunPowershellCmd(fmt.Sprintf("(Get-Service -Name %s -ErrorAction Stop).Status", service))
-		if err != nil {
-			t.Fatalf("Error getting service status: %v %v", err, out.Stderr)
-		}
-		status := strings.TrimSpace(out.Stdout)
-		out, err = utils.RunPowershellCmd(fmt.Sprintf("(Get-Service -Name %s -ErrorAction Stop).StartType", service))
-		if err != nil {
-			t.Fatalf("Error getting service starttype: %v %v", err, out.Stderr)
+	checkServices := func() []error {
+		var errs []error
+		for _, service := range services {
+			out, err := utils.RunPowershellCmd(fmt.Sprintf("(Get-Service -Name %s -ErrorAction Stop).Status", service))
+			if err != nil {
+				errs = append(errs, fmt.Errorf("Error getting service status: %v %v", err, out.Stderr))
```

Comment:
Nit: include `service` name in error message.

Reply:
Done.